testdata/p4_16_samples_outputs/pna-extract-local-header.p4-error

```diff
@@ -10,3 +10,12 @@ parser MainParserT<PM, MH, MM>(
 pna.p4(680): [--Wwarn=unused] warning: 'PM' is unused
 control MainControlT<PM, MH, MM>(
                      ^^
+pna-extract-local-header.p4(58): [--Wwarn=ignore-prop] warning: Result of AssignmentStatement is not defined: Error: Reading field from invalid header
+        hdr.h2.type1 = local_hdr.type1;
+                     ^
+pna-extract-local-header.p4(59): [--Wwarn=ignore-prop] warning: Result of AssignmentStatement is not defined: Error: Reading field from invalid header
```

Comment:
This error message does not make sense. Is it an error or a warning?

Reply:
These errors appeared when running the test on git, they are not on the local, I had to add